packages/@aws-cdk/aws-secretsmanager/lib/secret.ts

```diff
@@ -103,6 +103,20 @@ export interface SecretProps {
    */
   readonly secretName?: string;
 
+  /**
+   * **WARNING:** *It is **highly** encouraged to leave this field undefined and allow SecretsManager to create the secret value.
```

Comment:
Putting this here mucks with our doc generation. Put a caption as the first line. For example:

```
/**
  * Literal secret value (DANGEROUS!)
  */
```